src/main/java/bio/terra/cloudfiletodatastore/deltalayer/functions/MessageConverter.java

```diff
@@ -0,0 +1,15 @@
+package bio.terra.cloudfiletodatastore.deltalayer.functions;
+
+import bio.terra.cloudfiletodatastore.FileUploadedMessage;
+import com.google.events.cloud.storage.v1.StorageObjectData;
+
+public class MessageConverter {
```

Comment:
delete, if everything that calls this is also deleted

Reply:
I like having this logic separate so it can be tested apart from the GCF class.